src/Simplex_tree/concept/SimplexTreeOptions.h

```diff
@@ -29,5 +29,7 @@ struct SimplexTreeOptions {
   static constexpr bool contiguous_vertices;
   /// If true, the lists of `Node` with same label are stored to enhance cofaces and stars access.
   static const bool link_nodes_by_label;
+  /// If true, Simplex_handle will not be invalidated after insertions or removals.
+  static const bool simplex_handle_strong_validity;
```

Comment:
I was wondering if the name of the option should contain "stable" (or stability), but since this is related to iterator invalidation, validity seems fine.

Reply:
We could simply call it "stable_simplex_handles"? `simplex_handle_strong_validity` is the name Clément chose, so I kept it, but the other version would be shorter and still understandable.

Reply:
I am ok with either name.

Reply:
@VincentRouvreau Do you have an opinion on this?

Reply:
@hschreiber No strong opinion. Choose the one you prefer, as the name and the documentation is quite clear for me